releasetool/github.py

```diff
@@ -22,8 +22,11 @@
 
 import jwt
 import requests
-from cryptography.hazmat.backends import default_backend
 
+import cryptography
+from cryptography.hazmat.primitives import serialization
+
+CRYPTOGRAPHY_VERSION = cryptography.__version__
```

Comment:
This looks unused - can remove this and its import.

Reply:
Fixed in https://github.com/googleapis/releasetool/pull/532/commits/e14375e752a4caa5141f302c4baf21a1c39b6081